FogCreek/Glitch-Community
# circular-snout: pull live homepage and pupdates data from cms.glitch.me

## Links
* [circular-snout.glitch.me](https://circular-snout.glitch.me)

## Changes:
This pulls live homepage and pupdates data from cms.glitch.me instead of from the ~community filesystem.

- Remove `src/curated/{page.json}`
- Remove part of merge script that handles live updates to home.json
- Remove routes `/api/{pupdates|home}` and the helpers in `server/curated` that wrote pupdates.json and home.json to the filesystem

## How To Test:
* The homepage should load with all the content we expect
* Open pupdates, they should all load

## Feedback I'm looking for:
- How do we make sure cms.glitch.me is always online? Is it reasonable to assume that ~community will ping it for data often enough that it will never go to sleep? That feels risky.
- Does this approach seem sound?
